tests/TestData/Providers/Reflection/ReflectionClassesTestDataProviders.php

```diff
@@ -64,4 +64,17 @@ public static function finalClassesProvider(): ?Generator
             yield "class $class->name" => [$class];
         }
     }
+
+        public static function readonlyClassesProvider(): ?Generator
+    {
+        $classes = ReflectionStubsSingleton::getReflectionStubs()->getClasses();
+        $filtered = EntitiesFilter::getFiltered(
+            $classes,
+            null,
```

Comment:
If I'm not mistaken,  some closure that will filter out non-readonly classes should be here instead of `null`. like `fn(PhpClass $class) => $class->readonly === false`